docs/kb/semgrep-supply-chain/why-no-findings.md

```diff
@@ -0,0 +1,66 @@
+---
+description: Troubleshoot why findings for Semgrep Supply Chain are not showing.
+tags:
+  - Semgrep Supply Chain
+---
+# Why aren't Supply Chain findings showing?
```

Comment:
I assume (based on the content) that this referred to both reachable and unreachable findings - is this correct? And if so, do we want to mention it explicitly? Also, I assume this is applicable for CLI and SCP interfaces? Maybe we we want to mention that also.

Reply:
It's only applicable for SCP, so maybe I should specify. The reason I didn't is that non-SCP use of SSC is currently not considered a happy path, but I don't think it hurts to be clear.

This is all findings, yep. I could also change the title to say "any", maybe?

"Why aren't any Supply Chain findings showing in Semgrep Cloud Platform?" would be a possible revision.

Reply:
oh, wait a sec, I remember why I didn't specify. the reason is that a bunch of these possibilities apply to both - only the branch thing is specific to SCP.

Reply:
"Why aren't any Supply Chain findings showing in Semgrep Cloud Platform" sounds good!

Reply:
Ooh okay. So it's probably worth either adding that as a section, or being super clear upfront. I'd actually incline to the former, I'll go back and add that now.